test/codegen-generated-classes-test/src/main/resources/codegen-resources/autoconstructedlists/codegen.config

```diff
@@ -0,0 +1,5 @@
+{
+    "serviceInterfaceName" : "AmazonCodeGenerationJsonRpcCustomized",
+    "packageSuffix" : "protdofjafoobarbazocol.jsonrpc.customized",
```

Comment:
Lol what a name.

Reply:
LOL, I was trying to see if `packageSuffix` was supported but it doesn't seem to be atm.